coconut/control/control.go

```diff
@@ -200,6 +200,39 @@ func CreateEnvironment(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.C
 		return
 	}
 
+	if cmd.Flags().Changed("auto") {
```

Comment:
If I understand correctly that you're checking whether `auto` is set, `GetBool` seems more appropriate than `Changed`. The former returns the absolute value of the flag, the latter tells you whether a setting was passed that overrides any default.